### ISSUE TITLE
Finish web factional port

### ISSUE THREAD

**Issue by bethlakshmi** [bethlakshmi/gbe-divio-djangocms-python2.7#65]
I just pulled out a copy of the old DB, and found that there are some people who got into the old DB who are not into the new:
- 10 new users, although one seems to have left before making a profile.
- 1 Act application (draft mode)
- 1 Vendor application (also draft)
- 5 personas (one is a duplicate)
- some photos for those personas.

I figure I should port them, and am assuming it's relatively high.
Let me know if you want the contact info of the Vendor and Act appliers, in case you want to reach out personally - I can make their accounts & data move, but they have to reset their passwords.

All this data came in between Oct and Nov.  Once we ported the DNS, no new data came in.

**Comment by burlexpo** [comment]
Yeah, I'd like the contact info -- or at least to know who they are so I can look them up.

**Comment by bethlakshmi** [comment]
sent by email.

**Comment by bethlakshmi** [comment]
Started the import...  rows 5 & 6 from the spreadsheet I sent are imported...  6 left to go...

**Comment by bethlakshmi** [comment]
All importing done - including personas, act bid and vendor bid.
One user basically had a duplicate persona - I decided to only make one.
And the act/vendor bids were made against GBE 2018, I updated them to GBE 2020, since I believe this was the intent (and GBE 2020 wasn't on web factional).

With that done, I'll turn off web factional usage, and stop all billing!  Nothing else is on that server that I care about!!
YAY!
